hermes-management/src/main/java/pl/allegro/tech/hermes/management/domain/owner/CrowdOwnerSource.java

```diff
@@ -50,4 +55,8 @@ public Owner get(String id) {
                 .collect(Collectors.toList());
     }
 
+    @Override
+    public Optional<String> usageHint() {
+        return Optional.of("Crowd group (or groups separated by ',')");
```

Comment:
It's true. On the other hand, it's all in one place ;) Alternatively I could move these custom hints to console config json, what do you think?